tuner/examples/test/tuner_test.py

```diff
@@ -4,22 +4,95 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+import argparse
+from pathlib import Path
 from tuner import libtuner
 
 
+class TestTuner(libtuner.TuningClient):
+    def __init__(self):
+        self.compile_flags = ["--compile-from=executable-sources"]
+        self.benchmark_flags = ["--benchmark_repetitions=3", "--input=1"]
```

Comment:
Do we need the input argument?

Reply:
Benchmarks take a single input for the number of iterations to run afaict. I think we can add some bool option (i.e., is_benchmark or something) to the benchmark function that can add this input automatically, but I'll leave that as a followup.